startme.sh

```diff
@@ -6,13 +6,15 @@ SKIP_REFRESH=${SKIP_REFRESH:-}
 
 if [ -z "${OVERRIDE}" ]
 then
-	mkdir -p /var/www/html
-	darkhttpd /var/www/html --port 80 --daemon
-	if [ -z "${SKIP_REFRESH}" ]
-	then
-		wget -qO- https://codeload.github.com/dokku/simp_le/tar.gz/master | tar xz -C /
-	fi
-	/simp_le-master/simp_le.py --default_root /var/www/html "$@"
+    mkdir -p /var/www/html
+    darkhttpd /var/www/html --port 80 --daemon
+    if [ -z "${SKIP_REFRESH}" ]
+    then
+        cd /simp_le-master
+        git pull
```

Comment:
Ditto here, I feel as though we should always have a known "state" of the image, even if this means we need to conduct maintenance every so often.

Reply:
If we are going to grab a specific release tag when building the Docker image then I don't see a point keeping this code in. I'd like to remove the SKIP_REFRESH option, it's feature set, and associated documentation.

The main letsencrypt plugin already grabs the latest Docker image anyways on every run.

What do you think?

Reply:
Yeah lets remove SKIP_REFRESH then. In dokku's use-case, its not necessary or desired.